site-builder.js

```diff
@@ -116,7 +116,9 @@ SiteBuilder.prototype.build = function() {
 
 SiteBuilder.prototype.syncRepo = function() {
   this.logger.log('syncing repo:', this.repoName);
-  return this.spawn(this.git, ['pull']);
+  var that = this;
+  return this.spawn(this.git, ['pull'])
+    .then(function() { that.spawn(that.git, ['stash']); });
```

Comment:
I am still trying to grok promises, but when it's a bare function with no arguments and just one expression in it, can't you just do `.then(that.spawn(that.git, ['stash']))`?